assets/javascripts/discourse/components/ai-persona-editor.gjs

```diff
@@ -78,6 +112,12 @@ export default class PersonaEditor extends Component {
     }
   }
 
+  @action
+  toggleVisionEnabled() {
```

Comment:
As I say if you use the existing `toggleField` function this will become simpler

Reply:
sorted